### ISSUE TITLE
Do surveys in emails

### ISSUE THREAD

**Issue by neilkakkar** [PostHog/posthog#21071]
## Is your feature request related to a problem?

Sometimes, we don't want to wait for a user to come into the product to start a survey. Instead, send them the survey via email, and allow responding from within the email
*Please describe.*

## Describe the solution you'd like

A way to run surveys in emails / or at least pre-populate in email and finish on a custom landing page, all managed by posthog

**Comment by marandaneto** [comment]
This issue depends on https://github.com/PostHog/posthog/issues/27380

**Comment by lucasheriques** [comment]
@marandaneto not really, as it's possible to execute the survey [within the email only with AMP](https://amp.dev/documentation/guides-and-tutorials/start/create_email). so the forms can be used within the email client, not really depending on a full screen survey

**Comment by lucasheriques** [comment]
[+1 from this](https://posthog.slack.com/archives/C011L071P8U/p1741882235449459)

**Comment by jacksonsmith** [comment]
Hey @lucasheriques, great to see another engineer from Belo Horizonte making an impact out there!
Thanks for working on this feature — it’s going to be a big help for us here.

**Comment by lshaowei18** [comment]
Hi all, here's an example of an hosted survey: https://us.posthog.com/external_surveys/0198f82b-6c8a-0000-5c54-d645ba4d07ab

Been researching on jobs to be done theory and I thought it would be interesting to know what jobs people are hiring email surveys to do. 

(P.S I'm not part of the posthog team)
